core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -610,11 +610,34 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
       } yield ()
     }
 
+  def assertNoChildrenBlockingWorkspaceDeletion(workspace: Workspace): Future[Unit] = for {
+    workspaceChildren <- samDAO
+      .listResourceChildren(SamResourceTypeNames.workspace, workspace.workspaceId, ctx)
+      .map(
+        // a workspace may have a single child, if that child is the google project
```

Comment:
Can you add more detail in the comment about why it is still OK to delete in this case?